<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="l_spect_mean_range">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Spectral mean range</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_unit1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>rel. 1/cm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_min_mean_range">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_from">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_to">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_max_mean_range">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_unit2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>rel. 1/cm</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_scansize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Scan size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_sx_px">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_sy_px">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_sx_mu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_sy_mu">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_px">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>px</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_mu">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Âµm</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_x">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_y">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QComboBox" name="in_peak">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>TA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>G</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2D</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="l_initvals">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Initial values</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_int">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Intensity</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_lw">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Linewidth</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_init_int">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_init_lw">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_unit5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>rel. 1/cm</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_unit3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>arb. u.</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_thresholds">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Threshold intervals</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_min_int">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_min_pos">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_int2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Intensity</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_lw2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Linewidth</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_unit6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>arb. u.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_min_lw">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>480</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_unit7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>460</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>rel. 1/cm</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_unit8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>rel. 1/cm</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_pos2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_max">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_min">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_max_int">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_max_lw">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_max_pos">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>460</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_start_fit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>580</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start fit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_save_fit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>620</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save fit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="box_scan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>281</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Scan details</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="box_peak">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Peak details</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_save_plot">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>690</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_int3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Intensity</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_pos3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_lw3">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>0</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Linewidth</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_twistangle">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Twist angle</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_twistanglegrad">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>360</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Twist angle gradient</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>700</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Aaron SchÃ¤pers, 2023</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>360</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Spectrum</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>690</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_import_scan">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Raman scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_import_fit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>660</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load fit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_default_values">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Use default values</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view_int">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>291</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view_pos">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>291</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view_lw">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>20</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view_theta">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view_grad_theta">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>390</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view_spectrum">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>390</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <zorder>box_scan</zorder>
   <zorder>box_peak</zorder>
   <zorder>l_spect_mean_range</zorder>
   <zorder>l_unit1</zorder>
   <zorder>in_min_mean_range</zorder>
   <zorder>l_from</zorder>
   <zorder>l_to</zorder>
   <zorder>in_max_mean_range</zorder>
   <zorder>l_unit2</zorder>
   <zorder>l_scansize</zorder>
   <zorder>in_sx_px</zorder>
   <zorder>in_sy_px</zorder>
   <zorder>in_sx_mu</zorder>
   <zorder>in_sy_mu</zorder>
   <zorder>l_px</zorder>
   <zorder>l_mu</zorder>
   <zorder>l_x</zorder>
   <zorder>l_y</zorder>
   <zorder>in_peak</zorder>
   <zorder>l_initvals</zorder>
   <zorder>l_int</zorder>
   <zorder>l_lw</zorder>
   <zorder>in_init_int</zorder>
   <zorder>in_init_lw</zorder>
   <zorder>l_unit5</zorder>
   <zorder>l_unit3</zorder>
   <zorder>l_thresholds</zorder>
   <zorder>in_min_int</zorder>
   <zorder>in_min_pos</zorder>
   <zorder>l_int2</zorder>
   <zorder>l_lw2</zorder>
   <zorder>l_unit6</zorder>
   <zorder>in_min_lw</zorder>
   <zorder>l_unit7</zorder>
   <zorder>l_unit8</zorder>
   <zorder>l_pos2</zorder>
   <zorder>l_max</zorder>
   <zorder>l_min</zorder>
   <zorder>in_max_int</zorder>
   <zorder>in_max_lw</zorder>
   <zorder>in_max_pos</zorder>
   <zorder>b_start_fit</zorder>
   <zorder>b_save_fit</zorder>
   <zorder>b_save_plot</zorder>
   <zorder>l_int3</zorder>
   <zorder>l_pos3</zorder>
   <zorder>l_lw3</zorder>
   <zorder>l_twistangle</zorder>
   <zorder>l_twistanglegrad</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton</zorder>
   <zorder>b_import_scan</zorder>
   <zorder>b_import_fit</zorder>
   <zorder>b_default_values</zorder>
   <zorder>view_int</zorder>
   <zorder>view_pos</zorder>
   <zorder>view_lw</zorder>
   <zorder>view_theta</zorder>
   <zorder>view_grad_theta</zorder>
   <zorder>view_spectrum</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
